ckcp/openshift_dev_setup.sh

```diff
@@ -301,12 +302,26 @@ install_openshfit_pipeline() {
   echo -n "  - openshift-pipeline application: "
   kubectl apply -f "$GITOPS_DIR/$APP/$APP.yaml" --wait >/dev/null 2>&1
   argocd app wait pipelines-service tektoncd --timeout=90 >/dev/null 2>&1
+  echo "OK"
+}
 
+postcheck() {
   # Wait until CRDs are synced to KCP
-  while [ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl api-resources | grep -c tekton.dev)" != 8 ]
+  echo -n "  - Sync CRDs to KCP: "
+  local length
+  length=$(echo $TEKTON_CRDS | tr ',' '\n' | wc -l)
+  local resources_regexp
+  resources_regexp=$(echo $TEKTON_CRDS | tr ',' '|')
+  local  wait_period=0
+  while [[ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl api-resources -o name  2>&1 | grep -E "$resources_regexp" | wc -l )" -ne $length ]]
   do
+    wait_period=$((wait_period+10))
+    if [ $wait_period -gt 300 ];then
+      echo "Failed to sync resources to KCP"
```

Comment:
fixed. It will print the exace CRs  that's not synced. It looks like

```
[openshfit-pipeline]
  - openshift-pipeline application: OK
  - Sync CRDs to KCP: ..............................Failed to sync following resources to KCP:
    * repositories.pipelinesascode.tekton.dev1
```